bundle/trampoline/python_dbr_warning.go

```diff
@@ -60,11 +61,32 @@ func hasIncompatibleWheelTasks(ctx context.Context, b *bundle.Bundle) bool {
 		}
 
 		if task.ExistingClusterId != "" {
-			version, err := getSparkVersionForCluster(ctx, b.WorkspaceClient(), task.ExistingClusterId)
-			// If there's error getting spark version for cluster, do not mark it as incompatible
-			if err != nil {
-				log.Warnf(ctx, "unable to get spark version for cluster %s, err: %s", task.ExistingClusterId, err.Error())
-				return false
+			var version string
+			var err error
+			// If the cluster id is a variable and it's not resolved, it means it references a cluster defined in the same bundle.
+			// So we can get the version from the cluster definition.
+			// It's defined in a form of resources.clusters.<cluster_key>.spark_version while the value here is
+			// ${resources.clusters.<cluster_key>.id}
+			if strings.HasPrefix(task.ExistingClusterId, "${") {
```

Comment:
You don't need to have this check, you can directly use the `ok` value from `dynvar.PureReferenceToPath`

Reply:
Yeah, I know, just want to make the intention here a bit more clear when reading it